plugins/sc/qml/Keyboard_sc.qml

```diff
@@ -0,0 +1,92 @@
+/*
+ * Copyright 2013 Canonical Ltd.
+ *
+ * This program is free software; you can redistribute it and/or modify
+ * it under the terms of the GNU Lesser General Public License as published by
+ * the Free Software Foundation; version 3.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+import QtQuick 2.4
+import keys 1.0
+
+KeyPad {
+    anchors.fill: parent
+
+    content: c1
+    symbols: "languages/Keyboard_symbols.qml"
+
+    Column {
+        id: c1
+        anchors.fill: parent
+        spacing: 0
+
+        Row {
+            anchors.horizontalCenter: parent.horizontalCenter;
+            spacing: 0
+
+            CharKey { label: "q"; shifted: "Q"; extended: ["1"]; extendedShifted: ["1"]; leftSide: true; }
+            CharKey { label: "w"; shifted: "W"; extended: ["2"]; extendedShifted: ["2"] }
+            CharKey { label: "e"; shifted: "E"; extended: ["3","ê","è","é","ë","€"]; extendedShifted: ["3","Ê","È","É","Ë","€"] }
+            CharKey { label: "r"; shifted: "R"; extended: ["4"]; extendedShifted: ["4"] }
+            CharKey { label: "t"; shifted: "T"; extended: ["5","·ti"]; extendedShifted: ["5","·TI"] }
+            CharKey { label: "y"; shifted: "Y"; extended: ["6"]; extendedShifted: ["6"] }
+            CharKey { label: "u"; shifted: "U"; extended: ["7","û","ù","ú","ü"]; extendedShifted: ["7","Û","Ù","Ú","Ü"] }
+            CharKey { label: "i"; shifted: "I"; extended: ["8","î","ì","í","ï"]; extendedShifted: ["8","Î","Ì","Í","Ï"] }
+            CharKey { label: "o"; shifted: "O"; extended: ["9","õ","ô","ò","ó","ö","º","œ"]; extendedShifted: ["9","Õ","Ô","Ò","Ó","Ö","º","Œ"] }
+            CharKey { label: "p"; shifted: "P"; extended: ["0"]; extendedShifted: ["0"]; rightSide: true; }
+        }
+
+        Row {
+            anchors.horizontalCenter: parent.horizontalCenter;
+            spacing: 0
+
+            CharKey { label: "a"; shifted: "A"; extended: ["à","å","ã","â","á","ä","ª","æ"]; extendedShifted: ["À","Å","Ã","Â","Á","Ä","ª","Æ"]; leftSide: true; }
+            CharKey { label: "s"; shifted: "S"; extended: ["·si","ß","$"]; extendedShifted: ["·SI","ẞ","$"] }
+            CharKey { label: "d"; shifted: "D"; extended: ["·dd"]; extendedShifted: ["·DD"] }}
```

Comment:
There is an extra closing brace on line 53, which is causing the keyboard to fail to load.